lib/ebay_api/operations/sell/inventory/inventory_item_groups/delete.rb

```diff
@@ -0,0 +1,15 @@
+class EbayAPI
+  scope :sell do
+    scope :inventory do
+      scope :inventory_item_groups do
+        operation :delete do
+          path { inventory_item_group_key }
+          option :inventory_item_group_key
+          http_method :delete
+          response(204) { true }
+          response(404) { nil }
```

Comment:
Do we really want to ignore 404 in all situations? I think it is better to throw a special exception (something like `EbayAPI::NotFound` and let calling application to decide.

Reply:
It makes sense to me to not raise an exception if the object to be deleted does not exist. The end result is the same - the object does not exist.